rrfs-test/validated_yamls/templates/obtype_config/abi_g18.yaml

```diff
@@ -0,0 +1,488 @@
+                     name: GeoVaLs/ice_area_fraction
+                   minvalue: 0.99
+                 value: 20
+               - where:
+                 - variable:
+                     name: GeoVaLs/surface_snow_area_fraction
+                   minvalue: 0.99
+                 value: 15
+
+       # Criteria assignment for Thinning 
+       - filter: Variable Assignment
+         assignments:
+         - name: DerivedMetaData/thinningCriteria
+           type: int
+           function:
+             name: IntObsFunction/Arithmetic
+             options:
+               variables:
+               - name: DerivedMetaData/surfaceParam
+               coefs: [1]
+
+       # Thinning
+       - filter: Gaussian Thinning
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         horizontal_mesh: 60
+         use_reduced_horizontal_grid: true
+         distance_norm: maximum
+         time_mesh: 'PT06H'
+         time_min: '2024-05-26T22:30:00Z'
+         time_max: '2024-05-27T01:30:00Z'
+         priority_variable: DerivedMetaData/thinningCriteria
+         action:
+           name: reject
+
+       obs post filters:
+       # Regional domain check to remove data outside of model domain
+       - filter: Bounds Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         test variables:
+         - name: GeoVaLs/observable_domain_mask
+           flag all filter variables if any test variable is out of bounds: true
+           minvalue: 0.0
+           maxvalue: 0.5
+
+       # Satellite Zenith Angle Check
+       - filter: Bounds Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         test variables:
+         - name: MetaData/sensorZenithAngle
+         maxvalue: 65.
+         action:
+           name: reject
+
+       # from read_abi: Reject when cloud fraction is more than 30
+       - filter: Domain Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         where:
+         - variable:
+             name: MetaData/cloudAmount
+             channels: 8
+           maxvalue: 30.
+
+       # from read_abi: WV Channels 8-10 scene consistency check SDTB>1.3
+       - filter: Domain Check
+         filter variables:
+         - name: brightnessTemperature
+           channels: *abi_g18_channels
+         where:
+         - variable:
+             name: ClearSkyStdDev/brightnessTemperature
+             channels: 8-10
+           maxvalue: 1.3
+           max_exclusive: true
+
+       # Surface type check
+       - filter: RejectList
+         filter variables:
+         - name: brightnessTemperature
+           channels: 7, 11-16
+         where:
+         - variable:
+             name: GeoVaLs/land_area_fraction
+           minvalue: 0.99
+
+       - filter: RejectList
```

Comment:
Please add a comment here.

Reply:
> Please add a comment here.

added comments for each surface type check.